rds_to_s3/etl_lambda.py

```diff
@@ -37,9 +38,28 @@
 
 def setup_engine():
     """Set up SQLAlchemy engine."""
-    return create_engine(
-        f"postgresql+psycopg2://{os.getenv('DB_USERNAME')}:{os.getenv('DB_PASSWORD')}@{os.getenv('DB_HOST')}:{os.getenv('DB_PORT')}/{os.getenv('DB_NAME')}"
-    )
+    try:
```

Comment:
Very nice change, making the code more readable and implementing error handling!